WordPress/gutenberg
# FormToggle: do not use "/" math operator

## What?


Refactor `FormToggle`'s styles away from using the `/` math operator

## Why?


As flagged in https://github.com/WordPress/gutenberg/pull/56515#discussion_r1410184116, the `/` operator is deprecated in Sass

## How?


Using the `math` module and the `math.div` operator (as suggested [here](https://github.com/WordPress/gutenberg/pull/56515#discussion_r1410396390))

## Testing Instructions





- Make sure that `FormToggle` renders correctly in Windows High Contrast Mode
- Make sure that there are no deprecation warnings when building the plugin locally (as mentioned by @andrewserong [here](https://github.com/WordPress/gutenberg/pull/56515#discussion_r1410184116))
